tests/unit/test_exec.py

```diff
@@ -1,9 +1,17 @@
-__copyright__ = "Copyright (c) 2020-2021 Jina AI Limited. All rights reserved."
-__license__ = "Apache-2.0"
+import os
+
+from jina import Executor
+
+try:
+    from vggish_audio_encoder import VggishAudioEncoder
+except:
+    from jinahub.encoders.audio.vggish_audio_encoder import VggishAudioEncoder
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+def test_load():
+    encoder = Executor.load_config(os.path.join(cur_dir, '../../config.yml'))
```

Comment:
do we need more unit tests?

Reply:
Yes, most of the tests should be unit tests

Reply:
added the dimension test that Yongxuanzhang mentioned